webpack_loader/contrib/jinja2ext.py

```diff
@@ -1,9 +1,21 @@
 import jinja2.ext
 
-from ..templatetags.webpack_loader import render_bundle
+from ..templatetags.webpack_loader import *
+from ..contrib.pages.templatetags.webpack_pages import *
 
 
 class WebpackExtension(jinja2.ext.Extension):
     def __init__(self, environment):
         super(WebpackExtension, self).__init__(environment)
-        environment.globals["render_bundle"] = lambda *a, **k: jinja2.Markup(render_bundle(*a, **k))
+        environment.globals.update({
+            "render_bundle": render_bundle,
+            "render_entrypoint": render_entrypoint,
+            "register_entrypoint": jinja2.contextfunction(register_entrypoint),
+            "render_css": jinja2.contextfunction(render_css),
+            "render_js": jinja2.contextfunction(render_js),
+            "inline_entrypoint": inline_entrypoint,
+            "asset": jinja2.contextfunction(asset_url),
```

Comment:
This looks like a ton of new features that may not strictly be related to webpack 5. I'd recommend shipping the bare minimum required for webpack 5 support in this PR and add these features as individual PRs later.

//cc @fjsj

Reply:
Yes, this is not strictly related. That's why I put it into webpack_loader.contrib.
Should I change this PR to a separate branch on my fork, where I delete the contrib part?

Reply:
Yes, may be open a new PR that strictly adds webpack 5 support without any new features. Then open issues/PRs for each feature we want to add.